pkg/graph/schema.resolvers.go

```diff
@@ -16,37 +16,27 @@ import (
 	"github.com/cmsgov/easi-app/pkg/models"
 )
 
-func (r *accessibilityRequestResolver) Documents(ctx context.Context, obj *models.AccessibilityRequest) ([]*model.AccessibilityRequestDocument, error) {
-	files, fileErr := r.store.FetchFilesByAccessibilityRequestID(ctx, obj.ID)
+func (r *accessibilityRequestResolver) Documents(ctx context.Context, obj *models.AccessibilityRequest) ([]*models.AccessibilityRequestDocument, error) {
+	documents, documentsErr := r.store.FetchFilesByAccessibilityRequestID(ctx, obj.ID)
 
-	if fileErr != nil {
-		return nil, fileErr
+	if documentsErr != nil {
+		return nil, documentsErr
 	}
 
-	documents := []*model.AccessibilityRequestDocument{}
-
-	for i, file := range *files {
-		document := &model.AccessibilityRequestDocument{
-			ID:         file.ID,
-			RequestID:  file.RequestID,
-			Name:       fmt.Sprintf("Test Doc Number %+v", i+1),
```

Comment:
Removing the name assignment here changes the front end a bit until the doc type story gets played. Not a problem, but flagging it so we're aware.